core/arch/interp.c

```diff
@@ -244,8 +244,14 @@ init_build_bb(build_bb_t *bb, app_pc start_pc, bool app_interp, bool for_cache,
      * whose fall-through hits our hook.  We avoid interpreting our own hook
```

Comment:
This solves only the test hello world app right? Do any signal tests work? I assume no.

Reply:
This solves every single app crashing (i.e., every single test failing).  Some tests still fail for other reasons like many (not all) of the signal-related tests -- probably matching GA CI.